### PR TITLE
style(ui): copy update

### PR DESCRIPTION
**Pull request by brandonhaslegs**
Just wanted to go around and make some copy updates to make some things more casual and friendly. Also attempting to change all straight apostrophes to curly apostrophes (read more [here](https://smartquotesforsmartpeople.com/)), which is more typographically accurate.

### CODE REVIEW

File: ui/src/project.ts [hunk 1]
```diff
@@ -145,21 +145,22 @@ const projectNameConstraints = {
...
+      "Your project’s name has some characters that aren’t supported. You can only use basic letters, numbers, and the _ , - and . characters.",
   },
 };
 
 const projectDescriptionConstraints = {
   length: {
     maximum: 256,
     tooLong:
-      "Your project description should not be longer than 256 characters.",
+      "Whoa Shakespeare, your project’s description can’t be longer than 256 characters. Shorten it a bit!",
```
Comment (juliendonck) [inline ui/src/project.ts:163]: ❤️

**Comment by brandonhaslegs** [comment]
@rudolfs can you help me fix this PR at some point? 😇 The conflicts aren't complicated, but I'm not sure how to fix them.

**Comment by rudolfs** [comment]
> @rudolfs can you help me fix this PR at some point? 😇 The conflicts aren't complicated, but I'm not sure how to fix them.

@brandonhaslegs done. You should pull before making new changes.

**Comment by brandonhaslegs** [comment]
Build still failing. 🥴

**Comment by rudolfs** [comment]
You forgot to update some specs, I fixed it.

**Review by rudolfs — APPROVED** [review]

**Pull request merged**